.github/workflows/pipeline.yaml

```diff
@@ -0,0 +1,15 @@
+name: Pipeline
+
+on:
+  pull_request:
+    branches:
+      - main
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  analysis:
+    name: Analysis
+    uses: ./.github/workflows/analysis.yml
```

Comment:
not sure if this is a style consideration you would want adopted but the ./ here on line 15 is not needed. ./ here explicitly refers to the current directory but should function the same without it.